test-browser/tests/ballot.js

```diff
@@ -79,8 +79,8 @@ function runTests (browser, testData) {
       })
       .perform((client, done) => {
         console.log('delegate - transact (not payable)')
-        browser.testFunction('delegate - transact (not payable)', '0xd3cd54e2f76f3993078ecf9e1b54a148def4520afc141a182293b3610bddf10f',
-              '[vm] from:0xca3...a733c, to:Ballot.delegate(address) 0x692...77b3a, value:0 wei, data:0x5c1...4d2db, 0 logs, hash:0xd3c...df10f',
+        browser.testFunction('delegate - transact (not payable)', '0x057d54e2f76f3993078ecf9e1b54a148def4520afc141a182293b3610bd3ce2e',
```

Comment:
display has changed but the executed tx remain the same. So changing hash here will break test

Reply:
How should it be now? I live everything the same here, just add new lines to the log?

Reply:
the secon argument should remain the same

Reply:
hash is the 2nd argument

Reply:
So, if I understand correctly I should switch hash back to `hash:0xd3c...df10f`?

Reply:
yes the hash is the same as before everywhere , you just changed the UI.